FBTweak/FBTweak.h

```diff
@@ -86,6 +86,12 @@ typedef id FBTweakValue;
  */
 @property (readonly ,nonatomic) dispatch_block_t currentValue;
 
+/**
+ @abstract \c YES if the tweak is enabled, \c NO otherwise. Default is \c YES. If \c NO, the
+ \c currentValue block will not be called.
+ */
+@property (nonatomic) BOOL isEnabled;
```

Comment:
I think that `enabled` is enough.

Reply:
I thought about it too, but I think `isEnabled` is good actually, we use it quite a lot.

Reply:
left as is